server/sonar-db-dao/src/main/java/org/sonar/db/qualityprofile/QualityProfileMapper.java

```diff
@@ -73,9 +73,9 @@
 
   void deleteProjectAssociationByProfileKeys(@Param("profileKeys") Collection<String> profileKeys);
 
-  List<ProjectQprofileAssociationDto> selectSelectedProjects(@Param("profileKey") String profileKey, @Param("nameQuery") String nameQuery);
+  List<ProjectQprofileAssociationDto> selectSelectedProjects(@Param("organizationUuid") String organizationUuid, @Param("profileKey") String profileKey, @Param("nameQuery") String nameQuery);
 
-  List<ProjectQprofileAssociationDto> selectDeselectedProjects(@Param("profileKey") String profileKey, @Param("nameQuery") String nameQuery);
+  List<ProjectQprofileAssociationDto> selectDeselectedProjects(@Param("organizationUuid") String organizationUuid, @Param("profileKey") String profileKey, @Param("nameQuery") String nameQuery);
 
-  List<ProjectQprofileAssociationDto> selectProjectAssociations(@Param("profileKey") String profileKey, @Param("nameQuery") String nameQuery);
+  List<ProjectQprofileAssociationDto> selectProjectAssociations(@Param("organizationUuid") String organizationUuid, @Param("profileKey") String profileKey, @Param("nameQuery") String nameQuery);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Split this 194 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)